src/java_jstat

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python
+#
+# igcollect - jvm_jstat
+#
+# Copyright (c) 2017, InnoGames GmbH
+
+from time import time
+from argparse import ArgumentParser
+from subprocess import check_output
+
+
+def main():
+    args = vars(parse_args())
```

Comment:
You could keep it as an `object`.  It looks like you don't need it as a `dict` later.